requirements/edx/github.txt

```diff
@@ -44,7 +45,7 @@ git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 -e git+https://github.com/edx/opaque-keys.git@27dc382ea587483b1e3889a3d19cbd90b9023a06#egg=opaque-keys
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
 git+https://github.com/edx/i18n-tools.git@v0.1.3#egg=i18n-tools==v0.1.3
-git+https://github.com/edx/edx-oauth2-provider.git@0.5.6#egg=oauth2-provider==0.5.6
+git+https://github.com/edx/edx-oauth2-provider.git@f0d9c76b8694873673da06d0edefd0bc9ec4d4a9#egg=oauth2-provider==0.5.6
```

Comment:
@clintonb We are using latest hash from `edx-oauth2-provider`, are you OK with this?